src/modules/auth/auth.module.ts

```diff
@@ -68,6 +71,16 @@ export class AuthModule implements NestModule {
         { path: '/v1/login/siwe/initiate', method: RequestMethod.POST },
         { path: '/v1/login/siwe/verify', method: RequestMethod.POST }
       )
-      .forRoutes({ path: '/*', method: RequestMethod.ALL });
+      .forRoutes({ path: '/*', method: RequestMethod.ALL })
+      .apply((_req: Request, res: Response, next: NextFunction) => {
+        if (!this.configService.get('BLOCKNUM_AUTH_ENABLED')) {
+          res
+            .status(404)
+            .send(
+              'Authentication at this endpoint is disabled. Other authentication protocols may be available'
+            );
+        }
+      })
+      .forRoutes({ path: '/v1/login', method: RequestMethod.ALL });
```

Comment:
Would it be better to just have `POST` here as it is the only endpoint that we're trying to disable? Better to be precise, I think. For example, if we ever did implement a `GET` endpoint, it could be strange behaviour.
```suggestion
      .forRoutes({ path: '/v1/login', method: RequestMethod.POST });
```